storage/engine/mvcc.go

```diff
@@ -1634,9 +1634,16 @@ func MVCCIterate(ctx context.Context,
 				}
 				break
 			}
-
 		} else {
-			iter.Seek(MakeMVCCMetadataKey(metaKey.Key.Next()))
+			if buf.meta.IsInline() {
```

Comment:
Why does this make such a big difference in this benchmark's speed? The data being written in the bench test looks to be non-inline...